charts/sda-svc/templates/finalize-deploy.yaml

```diff
@@ -84,29 +84,34 @@ spec:
           value: {{ .Values.global.broker.vhost | quote }}
         - name: BROKER_SERVERNAME
           value: {{ .Values.global.broker.host | quote }}
+      {{- if and .Values.global.broker.ssl .Values.global.tls.enabled }}
         - name: BROKER_SSL
           value: "true"
         - name: BROKER_VERIFYPEER
           value: {{ .Values.global.broker.verifyPeer | quote }}
-      {{- if .Values.global.broker.ssl }}
         - name: BROKER_CACERT
-          value: {{ include "tlsPath" . }}/ca.crt
+          value: {{ include "tlsPath" . }}/{{ .Values.finalize.tls.caCert }}
         {{- if .Values.global.broker.verifyPeer }}
         - name: BROKER_CLIENTCERT
-          value: {{ include "tlsPath" . }}/finalize.crt
+          value: {{ include "tlsPath" . }}/{{ .Values.finalize.tls.certFile }}
         - name: BROKER_CLIENTKEY
-          value: {{ include "tlsPath" . }}/finalize.key
+          value: {{ include "tlsPath" . }}/{{ .Values.finalize.tls.keyFile }}
         {{- end }}
+      {{- else }}
+        - name: BROKER_SSL
+          value: "false"
```

Comment:
maybe we don't need to add this `BROKER_SSL` in if clause and we can set it to value of `Values.global.tls.enabled` ?
but then again i am also fine with having it explicit

or do the same as `dbSSLmode` and have it in `_helpers`

Reply:
That's probably safe to do since the chance of someone running this stack with TLS and not having TLS in the DB and MQ can be considered almost nil.